taxonomies/categories.txt

```diff
@@ -25112,6 +25371,27 @@ fr:Yaourts sans lactose
 it:Yogurt senza lattosio
 nl:Lactose-vrije yoghurt, Lactose-vrije yoghurts, Yoghurt zonder lactose, Yoghurts zonder lactose
 
+<en:Yogurts
+en:Yogurt type fermented flavoured milk with sugar with bifidus, Yogurt type flavoured dairy specialty with sugar with bifidus
+fr:Lait fermenté de type yaourt aromatisé sucré au bifidus, spécialité laitière de type yaourt aromatisé sucré au bifidus
+ciqual_food_code:en:19539
+ciqual_food_name:en:Fermented milk or dairy specialty, yogurt type, flavoured, with sugar, with bifidus
+ciqual_food_name:fr:Lait fermenté ou spécialité laitière type yaourt, aromatisé, sucré, au bifidus
+
+<en:Yogurts
+en:Yogurt type fermented milk with fruits with sugar with bifidus, Yogurt type dairy specialty with fruits with sugar with bifidus
```

Comment:
For this kind of CIQUAL categories, I think it makes more sense to find which of our existing categories matches.
e.g. for this one, I would just associate it with the existing en:Fruit yogurts

Reply:
Entendu, je remplace en:Yogurts par en:Fruit yogurts